rules_gapic/java/java_gapic_repositories.bzl

```diff
@@ -12,279 +12,28 @@
-        artifact = "com.google.auth:google-auth-library-credentials:%s" % _AUTH_VERSION,
-    )
-
-def io_opencensus_opencensus_api():
-    native.maven_jar(
-        name = "io_opencensus_opencensus_api",
-        artifact = "io.opencensus:opencensus-api:0.15.0",
-    )
-
-def io_opencensus_opencensus_contrib_grpc_metrics():
-    native.maven_jar(
-        name = "io_opencensus_opencensus_contrib_grpc_metrics",
-        artifact = "io.opencensus:opencensus-contrib-grpc-metrics:0.12.3",
-    )
-
-def com_google_code_gson_gson():
-    native.maven_jar(
-        name = "com_google_code_gson_gson",
-        artifact = "com.google.code.gson:gson:2.7",
-    )
-
-def com_google_guava_guava():
-    native.maven_jar(
-        name = "com_google_guava_guava",
-        artifact = "com.google.guava:guava:20.0",
-    )
-
-def com_google_code_findbugs_jsr305():
-    native.maven_jar(
-        name = "com_google_code_findbugs_jsr305",
-        artifact = "com.google.code.findbugs:jsr305:3.0.2",
-    )
-
-def com_google_api_api_common():
-    native.maven_jar(
-        name = "com_google_api_api_common",
-        artifact = "com.google.api:api-common:1.7.0",
-    )
-
-def org_threeten_threetenbp():
-    native.maven_jar(
-        name = "org_threeten_threetenbp",
-        artifact = "org.threeten:threetenbp:1.3.3",
-    )
-
-def com_google_api_grpc_grpc_google_iam_v1():
-    native.maven_jar(
-        name = "com_google_api_grpc_grpc_google_iam_v1",
-        artifact = "com.google.api.grpc:grpc-google-iam-v1:0.12.0",
-    )
-
-def com_google_api_grpc_proto_google_iam_v1():
-    native.maven_jar(
-        name = "com_google_api_grpc_proto_google_iam_v1",
-        artifact = "com.google.api.grpc:proto-google-iam-v1:0.12.0",
-    )
-
-def com_google_http_client_google_http_client():
-    native.maven_jar(
-        name = "com_google_http_client_google_http_client",
-        artifact = "com.google.http-client:google-http-client:%s" % _HTTP_CLIENT_VERSION,
-    )
-
-def com_google_http_client_google_http_client_jackson2():
-    native.maven_jar(
-        name = "com_google_http_client_google_http_client_jackson2",
-        artifact = "com.google.http-client:google-http-client-jackson2:%s" % _HTTP_CLIENT_VERSION,
-    )
-
-def com_fasterxml_jackson_core_jackson_core():
-    native.maven_jar(
-        name = "com_fasterxml_jackson_core_jackson_core",
-        artifact = "com.fasterxml.jackson.core:jackson-core:2.9.2",
-    )
+def java_gapic_repositories():
+#    _maybe(
+#        http_archive,
+#        name = "com_google_api_gax_java",
+#        urls = ["https://github.com/googleapis/gax-java/archive/v1.38.0.zip"],
+#        strip_prefix = "gax-java-1.38.0" % _gax_version,
+#    )
+
+    # TODO: switch to http_archive once the gax change is pushed
+    _maybe(
+        native.local_repository,
+        name = "com_google_api_gax_java",
+        path = "/usr/local/google/home/vam/_/projects/github/vam-google/gax-java"
```

Comment:
Can you make this an http_archive rule to your fork on github? Or is it not worth it, since it will change once the other PR is merged?

Reply:
I think it is not worth it. I will change to point to real stuff, once gax PR is is submitted.